src/main/scala/akka/contrib/ddata/ORMapState.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright © 2014-2016 Typesafe, Inc. All rights reserved.
+ * No information contained herein may be reproduced or transmitted in any form
+ * or by any means without the express written permission of Typesafe, Inc.
+ */
+
+package akka.contrib.ddata
```

Comment:
Thanks for catching this.

Now we have `akka.contrib.actor` which is the contrib for `akka.actor`, and similary `akka.contrib.cluster.ddata` which is the contrib for `akka.cluster.ddata`.

Is this ok?